gulpfile.js

```diff
@@ -2,15 +2,38 @@ const gulp = require('gulp');
 const babelify = require('babelify');
 const browserify = require('browserify');
 const source = require('vinyl-source-stream');
+const gutil = require('gulp-util');
+const notify = require('gulp-notify');
 
-gulp.task('default', ['css', 'browserify']);
+const reportError = function (error) {
+  notify({
+    title: 'Task Failed [browserify]',
+    message: 'See console.',
+    sound: 'Sosumi' // See: https://github.com/mikaelbr/node-notifier#all-notification-options-with-their-defaults
+  }).write(error);
+
+  gutil.beep();
+
+  // Pretty error reporting
+  let report = '';
+  const chalk = gutil.colors.white.bgRed;
+  const lineNumber = error.loc.line;
+
+  report += chalk('TASK:') + ' browserify\n';
+  report += chalk('PROB:') + ` ${error.message}\n`;
+  report += chalk('LINE:') + ` ${lineNumber}\n`;
+  console.error(report); // eslint-disable-line
```

Comment:
`console` calls should be fine in this file. You should remove this comment and just add an `/* eslint no-console: "off" */` directive to the top of the file.

Reply:
👍